app/bundles/LeadBundle/EventListener/TypeOperatorSubscriber.php

```diff
@@ -368,7 +368,7 @@ private function getStageChoices(): array
      */
     private function getCategoryChoices(): array
     {
-        return $this->makeChoices($this->categoryModel->getLookupResults('global'), 'title', 'id');
+        return $this->makeChoices($this->categoryModel->getLookupResults('global', '', 0), 'title', 'id');
```

Comment:
I didn't meant to develop any new feature like a typeahead. I was thinking something like
```suggestion
        return $this->makeChoices($this->categoryModel->getLookupResults('global', '', 10000), 'title', 'id');
```

Reply:
Yeh sorry I do sometimes unnecessarily dive 😆 I did come to this conclusion at the end. I'll update it to this later as definitely you're right it just cuts off that edge case that might break some weird installation.